packages/kit/src/runtime/server/utils.js

```diff
@@ -55,3 +55,40 @@ export function normalize_request_method(event) {
 	const method = event.request.method.toLowerCase();
 	return method === 'delete' ? 'del' : method; // 'delete' is a reserved word
 }
+
+/**
+ * @param {Error} error
+ * @param {(error: Error) => string | undefined} get_stack
+ */
+export function serialize_error(error, get_stack) {
+	return JSON.stringify(clone_error(error, get_stack));
+}
+
+/**
+ * @param {Error} error
+ * @param {(error: Error) => string | undefined} get_stack
+ */
+function clone_error(error, get_stack) {
+	const {
```

Comment:
I wonder if this will drop the error `frame` that is used to show where in the source code the error has occurred when there's a compilation error in dev mode

Reply:
not if it's enumerable